packages/audio-block/src/types.ts

```diff
@@ -0,0 +1,34 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { AppBridgeBlock, Asset } from '@frontify/app-bridge';
+
+export enum TextPosition {
+    Below = 'Below',
+    Above = 'Above',
+}
+
+export type BlockSettings = {
+    positioning: TextPosition;
+    audio?: Asset;
+    description?: string;
+    title?: string;
+    attachments: { id: number }[];
```

Comment:
```suggestion
```
Shouldn't be part of settings
```ts
export type BlockAssets = {
	attachments?: Asset[]
	audio?: Asset
}
```
And use it with `useBlockAssets`

Reply:
removed attachments from the BlockSettings type.
But it don't think this BlockAssets type is helpful, because we can't use the useBlockAssets in a generic way like for the blockSettings:

This is not possible:

`useBlockAsssets<BlockAssets>(appBridge)`

If its not that what you suggested, then please hit me up